### PR TITLE
Shapes div solver tool

### PR DESCRIPTION
**Pull request by II245**

### CODE REVIEW

File: tools.py [hunk 1]
```diff
@@ -509,17 +511,58 @@ def split(
...
+python tools.py find_solution_for_frontend_input_output_shapes_divisibility --start 119 --end 121 --input-shape-restriction 16 --output-shape-restriction 32
+'''
+
+def find_solution_for_frontend_input_output_shapes_divisibility(
+		window_size,
+		window_stride,
+		sample_rate,
+		start,
+		end,
+		input_shape_restriction,
```
Comment (vadimkantorov) [inline tools.py]: please change the arg name to something more specific, e..g `input_time_dim_multiple`

File: tools.py [hunk 2]
```diff
@@ -509,17 +511,58 @@ def split(
...
+		output_shape_restriction
```
Comment (vadimkantorov) [inline tools.py]: same as above

File: tools.py (hunk 1 above)
Comment (II245) [reply tools.py]: done

File: tools.py (hunk 2 above)
Comment (II245) [reply tools.py]: done

**Pull request closed without merging**